<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settng_Others</class>
 <widget class="QWidget" name="Settng_Others">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color:black;
}
QLabel
{
font:14px;
}
QPushButton
{
border:1px solid white;
color:white;
background-color:black;
}
QLineEdit
{
color:black;
background-color:white;
}</string>
  </property>
  <widget class="QLabel" name="LBL_Border1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>231</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Border2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>451</width>
     <height>311</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Tips2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
color:rgb(255,255,0);</string>
   </property>
   <property name="text">
    <string>当前模式：～模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Tips1">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>162</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>冬夏模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_NormalMode">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>210</y>
     <width>99</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16px;color: rgb(192,192,192);background-color: rgb(0, 0, 0);border-top: 2px solid white;border-left: 2px solid white;</string>
   </property>
   <property name="text">
    <string>普通模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SummerMode">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>260</y>
     <width>99</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16px;color: rgb(192,192,192);background-color: rgb(0, 0, 0);border-top: 2px solid white;border-left: 2px solid white;</string>
   </property>
   <property name="text">
    <string>夏季模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num8">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>8</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>240</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>2</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>3</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Save">
   <property name="geometry">
    <rect>
     <x>347</x>
     <y>374</y>
     <width>99</width>
     <height>35</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>11</string>
   </property>
   <property name="text">
    <string>存储</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num9">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>9</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num7">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>7</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Tips4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,0);
font:14px;</string>
   </property>
   <property name="text">
    <string>请输入牵引货物吨位：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num4">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>240</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>4</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num0">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>0</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_InputWeight">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>5</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Clear">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>10</string>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Num6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="whatsThis">
    <string>6</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Tips3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>162</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>牵引货物：</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_WeightValue">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>151</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid white;
color:rgb(255,255,0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>LBL_Border1</zorder>
  <zorder>LBL_Border2</zorder>
  <zorder>LBL_Tips2</zorder>
  <zorder>LBL_Tips1</zorder>
  <zorder>BTN_NormalMode</zorder>
  <zorder>BTN_SummerMode</zorder>
  <zorder>Button_Num8</zorder>
  <zorder>Button_Num2</zorder>
  <zorder>Button_Num3</zorder>
  <zorder>Button_Num1</zorder>
  <zorder>Button_Save</zorder>
  <zorder>Button_Num9</zorder>
  <zorder>Button_Num7</zorder>
  <zorder>LBL_Tips4</zorder>
  <zorder>Button_Num4</zorder>
  <zorder>Button_Num0</zorder>
  <zorder>Button_Num5</zorder>
  <zorder>Button_Clear</zorder>
  <zorder>Button_Num6</zorder>
  <zorder>LBL_Tips3</zorder>
  <zorder>LBL_WeightValue</zorder>
  <zorder>Edit_InputWeight</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
